common/src/main/java/com/wynntils/wynn/objects/SpellType.java

```diff
@@ -156,6 +157,7 @@ public static SpellType forClass(ClassType classRequired, int spellNumber) {
                 return spellType;
             }
         }
+        WynntilsMod.error("SpellType not found for " + classRequired + " | Spell: " + spellNumber);
         return null;
```

Comment:
I.e. we should prevent this being reachable and throw an illegal argument exception if we do somehow reach it.